downstream/modules/platform/proc-controller-image-pull-policy.adoc

```diff
@@ -0,0 +1,26 @@
+[id="proc_controller-image-pull-policy_{context}"]
+
+= controller-image-pull-policy
```

Comment:
Is the title of this procedure intentional? Per modularization standards, we should begin procedure modules with a gerund. For example, we could say "Configuring your automation controller image pull policy". I'm just guessing. I have no idea what the task is here, but it should say that clearly in the title instead of using the file name as the title.